python/railjson_generator/railjson_generator/schema/simulation/train_schedule.py

```diff
@@ -26,6 +37,31 @@ def add_stop(self, *args, **kwargs):
         self.stops.append(stop)
         return stop
 
+    def add_allowance(self, *args, engineering: bool = False, **kwargs):
+        """Add an allowance to the train schedule. For more information on allowances, see
+        the documentation of the Allowance class in osrd_schemas."""
+        if engineering:
+            allowance = EngineeringAllowance(*args, **kwargs)
+        else:
+            allowance = StandardAllowance(*args, **kwargs)
+        self.allowances.append(allowance)
+
+    def add_standard_single_value_allowance(self, value_kind: str, value: float, distribution: str = "LINEAR"):
+        """Add a standard allowance with a single value. For more information on allowances, see
+        the documentation of the Allowance class in osrd_schemas."""
+        if value_kind == "time":
+            value = AllowanceTimeValue(seconds=value)
+        elif value_kind == "time_per_distance":
+            value = AllowanceTimePerDistanceValue(minutes=value)
+        elif value_kind == "percent":
```

Comment:
Might be better to stay coherent with the rest of the code which uses `"percentage"` (see AllowancePercentValue). Or use `percent` everywhere.